sdk/nodejs/src/client.ts

```diff
@@ -296,6 +303,27 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    private trackSDKConfigEvent(
+        url: string,
+        responseTimeMS: number,
+        res: Response,
+    ): void {
+        this.eventQueue.queueEvent(
+            DVCPopulatedUserFromDevCycleUser({ user_id: this.clientUUID }),
```

Comment:
Don't have a `user_id` when the config is fetched, so just using the clientUUID as the user_id...

Reply:
@ajwootto are you good with the event format here?

Reply:
I think we use the hostname as the user_id on Go. We need to decide and be consistent about it in these events

Reply:
@JamieSinn is that the case for the new `sdkConfig` event? I think we learned from the incident that the hostname isn't the most reliable, we could potentially combine them together.

Reply:
we definitely use hostname as the aggregation parameter. We can refactor it to use the UUID - but I definitely use the UUID for the sdkConfig events.

Reply:
should we do `${hostname}_${uuid}`? or just uuid?

Reply:
I would do `hostname_uuid` so we can give the customer something that correlates to their internals.

Reply:
changing the hostname that's passed into the bucketing logic to also be `hostname_uuid` to change the ones used for agg events, let me know if you think that's an issue.

Reply:
for agg events I think that's fine

Reply:
Actually - it might be easier to say `uuid@hostname` - to show that it's a given instance on a hostname? thoughts?

Reply:
yea that's a better format, also going to need to update the test-harness again...

Reply:
gotta keep us honest :P